pycarol/data_models/data_models.py

```diff
@@ -695,9 +695,12 @@ def _check_entity_template_types(self):
     def _check_dm_name(self):
 
         est_ = DataModel(self.carol)
-        est_.get_by_name(self.dm_name)
-        if est_.entity_template_ is not None:
-            raise Exception('mdm name {} already exist'.format(self.dm_name))
+        try:
+            est_.get_by_name(self.dm_name)
+        except Exception:
+            return
```

Comment:
it is not a good practice to catch all Exception at once. I understand your intent here... but I think that define a specific Exception to handle gonna be better and get future debugging easier.